src/lib/actions/RulesAction.ts

```diff
@@ -27,7 +27,12 @@ export class RulesAction {
 		const core: CodeAnalyzer = new CodeAnalyzer(config);
 
 		const enginePlugins = this.dependencies.pluginsFactory.create();
-		const addEnginePromises: Promise<void>[] = enginePlugins.map(e => core.addEnginePlugin(e));
+		const enginePluginModules = config.getCustomEnginePluginModules()
+			.map(pluginModule => require.resolve(pluginModule, {paths: [process.cwd()]})); // TODO: Remove this line as soon as it is moved to the core module.
```

Comment:
I just realized that I forgot to add in {paths: [process.cwd()]} on the core side. So doing it here for now. But soon i'll add it to core so you won't need to do this map to resolve the paths.